src/main/java/fr/uge/lp/databackbonepoc/linkresearch/configuration/EntitiesConfiguration.java

```diff
@@ -40,26 +40,28 @@ public record EntitiesConfiguration(
    */
   static EntitiesConfiguration fromResearchConfiguration(
       Research research,
-      List<MatcherConfiguration> matcherConfiguration,
+      List<MatcherConfiguration> matcherConfigurations,
       DBFetcher leftEntityFetcher,
       DBFetcher rightEntityFetcher
   ) {
-    var leftFields = matcherConfiguration.stream()
-        .map(config -> config.getLeftEntityAttribute().getName())
+    var leftAttributes = matcherConfigurations.stream()
+        .map(MatcherConfiguration::getLeftEntityAttribute)
+        .distinct()
         .toList();
```

Comment:
```suggestion
        .toSet();
```

Est-ce qu'on aurait pas le même comportement comme ça ?